gui/theme.tcl

```diff
@@ -167,5 +167,322 @@ namespace eval ttk::theme::imunes {
+	    -lightcolor [list pressed $colors(-darker)] \
+	    -darkcolor [list pressed $colors(-darker)] \
+	    ;
+
+	ttk::style configure TCheckbutton \
+	    -indicatorbackground "#ffffff" \
+	    -indicatormargin {1 1 4 1} \
+	    -padding 2 ;
+	ttk::style configure TRadiobutton \
+	    -indicatorbackground "#ffffff" \
+	    -indicatormargin {1 1 4 1} \
+	    -padding 2 ;
+	ttk::style map TCheckbutton -indicatorbackground \
+	    [list  disabled $colors(-frame)  pressed $colors(-frame)]
+	ttk::style map TRadiobutton -indicatorbackground \
+	    [list  disabled $colors(-frame)  pressed $colors(-frame)]
+
+	ttk::style configure TMenubutton \
+	    -width -11 -padding 5 -relief raised \
+	    -fieldbackground #1b1f20 \
+        -foreground #F5F5F5
+
+	ttk::style configure TEntry -padding 1 -insertwidth 1 \
+	    -fieldbackground #1b1f20 \
+        -foreground #F5F5F5
+	ttk::style map TEntry \
+	    -background [list  readonly $colors(-frame)] \
+	    -bordercolor [list  focus $colors(-selectbg)] \
+	    -lightcolor [list  focus "#6f9dc6"] \
+	    -darkcolor [list  focus "#6f9dc6"]
+	    ;
+	# Modifi aqui
+	ttk::style configure TCombobox -padding 1 -insertwidth 1 -borderwidth 0 \
+	    -fieldbackground #1b1f20 \
+        -foreground #F5F5F5
+	ttk::style map TCombobox \
+	    -background [list active $colors(-lighter) \
+			     pressed $colors(-lighter)] \
+	    -fieldbackground [list {readonly focus} $colors(-selectbg) \
+				  readonly $colors(-frame)] \
+	    -foreground [list {readonly focus} $colors(-selectfg)] \
+	    ;
+	ttk::style configure ComboboxPopdownFrame \
+	    -relief solid -borderwidth 0
+	# modif aqui
+	ttk::style configure TSpinbox -arrowsize 15 -padding {2 0 10 0} -borderwidth 0 \
+		-fieldbackground #1b1f20 \
+        -foreground #F5F5F5 \
+        -background #ffffff
+    ttk::style map TSpinbox -background [list  readonly $colors(-frame)] \
+            -arrowcolor [list disabled $colors(-disabledfg)]
+
+	ttk::style configure TNotebook.Tab -padding {6 2 6 2} -borderwidth 0
+	ttk::style map TNotebook.Tab \
+	    -padding [list selected {6 4 6 2}] \
+	    -background [list selected $colors(-frame) {} $colors(-darker)] \
+	    -lightcolor [list selected $colors(-lighter) {} $colors(-dark)] \
+	    ;
+
+	# Treeview:
+	ttk::style configure Heading \
+	    -font TkHeadingFont -relief raised -padding {3}
+	ttk::style configure Treeview -background $colors(-otrocolor4) \
+	-fieldbackground $colors(-otrocolor4);
+	ttk::style map Treeview \
+	    -background [list selected $colors(-selectbg)] \
+	    -foreground [list selected $colors(-selectfg)] ;
+
+	ttk::style configure TLabelframe \
+	-borderwidth 2 -relief groove 
+
+	ttk::style configure TProgressbar -background $colors(-frame)
+
+	ttk::style configure Sash -sashthickness 6 -gripcount 10
+	   
+    # tk widgets.
+    ttk::style map Menu \
+        -background [list active $colors(-lighter)] \
+        -foreground [list disabled $colors(-disabledfg)]
+    
```

Comment:
I get an error for missing closing brace }, I guess it should be placed in this line.